api/organization/get-donation-ledger.js

```diff
@@ -1,10 +1,16 @@
 const S = require('fluent-schema')
-const { OrgDonationLedger, OrganizationId } = require('../../types/BillingInfo')
+const { OrgDonationLedger, OrganizationId } = require('../../types/Organization')
 const { Base200WithData } = require('../../common/reply')
 
 module.exports = {
-  querystring: S.object().prop('organizationId', OrganizationId).required(),
+  querystring: S.object()
+    .prop('organizationId', OrganizationId).required()
+    .prop('sizeRequest', S.boolean())
+    .prop('offset', S.integer().minimum(0))
+    .prop('limit', S.integer().minimum(1)),
```

Comment:
should this also pass noAuth since we don't require auth?

Reply:
nvm

Reply:
noAuth is a flag the caller can pass to /organization/:orgId if they want to be treated as "unauthed". i don't see how it makes sense here though since the ledger is either public and requires no auth or isn't public and does require auth.